conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/BodySerDe.java

```diff
@@ -19,15 +19,34 @@
 import io.undertow.server.HttpServerExchange;
 import java.io.IOException;
 import java.io.InputStream;
+import java.util.Optional;
 
 /** Request and response Deserialization and Serialization functionality used by generated code. */
 public interface BodySerDe {
```

Comment:
Do we also want to provide an `Endpoint` reference when directly calling the serde methods for binary responses:

https://github.com/palantir/conjure-java/blob/d585a57a01e68b4a99b3374137d630f3b3d0b5e2/conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/BodySerDe.java#L58

https://github.com/palantir/conjure-java/blob/d585a57a01e68b4a99b3374137d630f3b3d0b5e2/conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/BodySerDe.java#L73

Reply:
I don't think we need to update those